2021/38xxx/CVE-2021-38892.json

```diff
@@ -1,18 +1,100 @@
 {
-    "data_type": "CVE",
-    "data_format": "MITRE",
-    "data_version": "4.0",
-    "CVE_data_meta": {
-        "ID": "CVE-2021-38892",
-        "ASSIGNER": "cve@mitre.org",
-        "STATE": "REJECT"
```

Comment:
@ScottMooreIBM we received and processed a request from you (Request #1215258) stating that this ID be rejected as not a vulnerability. Is this change intentional?